pkg/operator/ceph/file/mds/mds.go

```diff
@@ -128,8 +128,11 @@ func (c *Cluster) Start() error {
 		}
 	}()
 
-	// Always create double the number of metadata servers to have standby mdses available
-	replicas := c.fs.Spec.MetadataServer.ActiveCount * 2
+	replicas := c.fs.Spec.MetadataServer.ActiveCount
```

Comment:
have we tested how cephfs in ceph will behave to it?

Reply:
When the standby mds is removed, it does result in a health warning:
```
    health: HEALTH_WARN
            insufficient standby MDS daemons available
```
The user could choose to suppress the error if they want.